src/main/java/com/exasol/adapter/document/documentfetcher/dynamodb/SerializableSqlNodeWrapper.java

```diff
@@ -62,17 +62,4 @@ private void readObject(final ObjectInputStream in) throws IOException, ClassNot
                     .parseExpression(sqlNodeAsJson);
         }
     }
-
-    /**
-     * Custom deserialization.
-     * <p>
-     * This method is called during deserialization
-     * </p>
-     * 
-     * @throws ObjectStreamException since no data is available if this method is called
-     */
-    private void readObjectNoData() throws ObjectStreamException {
-        throw new InvalidObjectException(
-                ExaError.messageBuilder("E-VS-DY-34").message("Failed to deserialize SqlNode.").toString());
-    }
```

Comment:
Better keep this method: https://stackoverflow.com/questions/7445217/java-when-to-add-readobjectnodata-during-serialization

Reply:
Re-added  with `@SuppressWarnings("unused")` annotation and additional comment.